services/api-docs/Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM node:12-alpine
-ARG NODE_ENV=production
 
-RUN apk add --no-cache bash git python make g++ bash
+ARG NODE_ENV=production
 
-COPY ./package.json /service/package.json
-COPY ./yarn.lock /service/yarn.lock
+# Install app dependencies
+COPY package.json /service/package.json
+COPY yarn.lock /service/yarn.lock
 
-RUN cd /service; yarn install --frozen-lockfile
+# Update & install required packages
+RUN apk add --update bash git make python g++ curl; \
+  cd /service; \
+  yarn install --frozen-lockfile; \
+  apk del python make g++
```

Comment:
why does this need to be one command now?

Reply:
So this ensures that its all done in one layer. 
The `apk del python make g++` will actually matter otherwise its pointless.

The downside might be that it might not be so well cached, as everything is one layer, so building it might take a slight bit longer, but i think time will be saved when comes to uploading docker image (as 200mb) is after all a fair amount.

I am wondering if we even need `apk add --update bash git make python g++ curl ` gonna try that out tonight

Reply:
ahh ok I see... I just don't like it cuz it's ugly but guess there's no way around that :)